bounded-collections/src/lib.rs

```diff
@@ -51,6 +51,59 @@ impl<T: Default> Get<T> for () {
 	}
 }
 
+/// Provides type conversion for [`Get<T>`] implementations using [`Into`]
+///
+/// This extension trait enables ergonomic conversion of values obtained through
+/// the [`Get<T>`] trait to different types via Rust's standard [`Into`] conversion.
+///
+/// # Usage Pattern
+/// The trait serves two primary purposes:
+/// 1. Add conversion capabilities to existing [`Get<T>`] implementations
+/// 2. Enable method chaining for type conversions
+///
+/// # Type Parameters
+/// - `T`: Source type being converted from (inferred from [`Get<T>`] implementation)
+/// - `I`: Target type being converted into (must be specified explicitly)
+/// # Examples
+/// ```
+/// use bounded_collections::Get;
+/// use bounded_collections::GetInto;
+///
+/// struct Foo;
+/// impl Get<u32> for Foo {
+/// 	fn get() -> u32 { 42 }
+/// }
+/// struct Bar;
+/// impl Get<u16> for Bar {
+///    fn get() -> u16 { 42 }
+/// }
+/// fn main() {
+/// let x: u32 = Bar::get_into();
+/// }
+/// ```
+pub trait GetInto<T, I> {
```

Comment:
Why did you changed this? Also this would not help for the downstream use case.

The only way this could work is when we also change the actual type bounds of the bounded collectives.

Reply:
> @Doordashcon please format the code :)

It's a misunderstanding, thought you meant

Reply:
cargo +nightly fmt 🙏